.github/release.yml

```diff
@@ -0,0 +1,23 @@
+# .github/release.yml
+# Modified from https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes
+# and https://gitmoji.dev/
+changelog:
+  exclude:
+    labels:
+      - changelog:ignore-for-release
+  categories:
+    - Added: ✨ New Features
+      labels:
+        - changelog:new-feature
+    - Changed: ♻️ Changed and Improved Functionalities 
+      labels:
+        - changelog:enhancement
+        - changelog:refactor
+    - Fixed: 🐛 Bugfixes and Security Improvements
+      labels:
+        - changelog:bugfix
+        - changelog:security
+    - Removed: 🔥 Removed and Deprecated Features
+      labels:
+        - changelog:removal
+        - changelog:deprecation
```

Comment:
You might add a catch-all for the remaining features I think. just in case.
I guess we will anyway have to manually re-order some of them, or the ones where the labels are missing which I can already tell that will happen.

Reply:
I agree on the ``*`` for the remaining

Reply:
this catchall "*" feature is not documented, did you come up with it or are you aware of some working undocumented functionalities? :D See  https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes

I might try it later